data/json/ammo_effects.json

```diff
@@ -115,18 +120,88 @@
   {
     "id": "EXPLOSIVE_RAUFOSS",
     "type": "ammo_effect",
-    "explosion": {
-      "power": 2.4,
-      "distance_factor": 0.6,
-      "fire": true,
-      "shrapnel": { "casing_mass": 28, "fragment_mass": 1.4, "recovery": 0, "drop": "null" }
-    }
+    "explosion": { "power": 2.4, "shrapnel": { "casing_mass": 28, "fragment_mass": 14 } }
   },
   {
     "id": "FRAG",
     "type": "ammo_effect",
     "explosion": { "power": 185, "shrapnel": { "casing_mass": 212, "fragment_mass": 0.05 } }
   },
+  {
+    "id": "20x66_FRAG",
+    "type": "ammo_effect",
+    "explosion": { "power": 40, "shrapnel": { "casing_mass": 15, "fragment_mass": 0.15 } }
+  },
+  {
+    "id": "20x66_EXPLOSIVE",
```

Comment:
TBH, these should be migrated/obsoleted, not updated.
They're too small to deliver a useful payload, and explosive small arms ammunition is quite firmly a 'war crime' sort of thing, so it's a non-starter for even being a prototype.

Reply:
I'm not entirely against, but i'm not 100% on rivtechs role and in how much these things are for fighting nether creatures, and if this PR merges they will at least have the proper ammo effect when/if they get adopted by aftershock.

But more importantly, i won't have time to replace their spawns in the loot pools with appropriate alternatives everywhere and i don't want to obsolete it if i'm going to half ass it.